### DISCUSSION TITLE
Possible to get multiple glyphs if multiple buttons are bound to the same action?

### DISCUSSION THREAD

**Discussion by Ansoneh** [Orange-Panda/Rewired-Glyphs#11] [Ideas]
I have some actions that have multiple keybindings. Would love to be able to get the glyphs for each different key.

**Comment by Orange-Panda** [comment]
Is this a case where you have modifier keys (such as "Shift+A") or is this a case where an action has multiple inputs mapped to it (such as Move Left being "A" and "Left Arrow" on keyboard)?

**Reply by Ansoneh** [reply]
It's the second case. Multiple keys/buttons that do the same thing.

**Reply by Orange-Panda** [reply]
This isn't currently possible with the InputGlyphs API but I will look into adding this functionality. 

It will likely be its own separate method that returns a set of all appropriate glyphs for that action and device.

**Reply by Ansoneh** [reply]
That's amazing. Thanks!

**Reply by Orange-Panda** [reply]
This feature has been implemented on the development branch and is planned to be included in the next release. It will return a list of *all* glyphs for an action for a particular player across all controller types. You may choose to filter this list by the Glyph's controller type if you only want to represent a single controller type.

It is still undergoing some testing so I'll let you know when it reaches a stable release after it is confirmed ready. If your implementation is really urgent you may choose to download the experimental development version but I'd recommend waiting for a stable release otherwise.

**Reply by Ansoneh** [reply]
Wow that was quick! Thanks a ton!

**Reply by Orange-Panda** [reply]
This feature was released in [2.0.0](https://github.com/Orange-Panda/Rewired-Glyphs/discussions/13). Check out [the documentation](https://orange-panda.github.io/Rewired-Glyphs/api/LMirman.RewiredGlyphs.InputGlyphs.GetGlyphSet.html) for more information on its usage
